.ci/setup_env_fedora.sh

```diff
@@ -17,7 +17,7 @@ sudo -E dnf -y install moreutils
 
 if [ "${TEST_CGROUPSV2}" == "true" ]; then
 	echo "Install podman"
-	sudo -E dnf -y install podman
+	sudo -E dnf -y install podman-1.6.2
```

Comment:
instead of hardcoding the version, can you add it to https://github.com/kata-containers/tests/blob/master/versions.yaml and then retrieve it using `get_test_version`?